wordpress-mobile/WordPress-iOS
# Wires site_is_atomic from the reader endpoint.

Integrates the changes from https://github.com/wordpress-mobile/WordPressKit-iOS/pull/240, so that we can also integrate the changes in D41085 into the App.

Bottom line is: the reader endpoint will now return a `site_is_atomic` flag in posts from atomic sites.  This PR wires that new flag into WPiOS.

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/13823

## To test:

1. Follow a private atomic site in the reader
2. Publish a post in that site, including an image just to make it more interesting.
3. Go to Reader > Followed Sites > Managed  > Pick your Atomic Private site.

Check that posts and images are visible.

## PR submission checklist:
